spec/uploaders/simple_forms_api/form_remediation/uploader_spec.rb

```diff
@@ -7,42 +7,154 @@
 module SimpleFormsApi
   module FormRemediation
     RSpec.describe Uploader do
```

Comment:
You can resolve the Metrics/ModuleLength cop like this


```ruby
RSpec.describe SimpleFormsApi::FormRemediation::Uploader do
```

Reply:
@stevenjcumming I went ahead and took care of this and all other issues. Would you be willing/able to take another look at your earliest convenience?